README.md

```diff
@@ -93,10 +93,10 @@ Stake your funds and create our validator.
 MAIN_VALIDATOR_NODE="tcp://testnet.palomaswap.com:26657"
 CHAIN_ID="paloma-testnet-4"
 DEFAULT_GAS_AMOUNT="10000000"
-VALIDATOR_STAKE_AMOUNT=100000grain
+VALIDATOR_STAKE_AMOUNT=100000000000ugrain
 PUBKEY="$(palomad tendermint show-validator)"
 palomad tx staking create-validator \
-      --fees 10000grain \
+      --fees 10000000000ugrain \
```

Comment:
@taariq Should this line just be removed?

Reply:
Yes. We can remove it.